master-server/src/main/java/com/replicated_log/master_server/service/MasterClient.java

```diff
@@ -0,0 +1,23 @@
+package com.replicated_log.master_server.service;
+
+import com.replicated_log.master_server.model.Item;
+import org.springframework.web.reactive.function.client.WebClient;
+
+public class MasterClient {
```

Comment:
perhaps we can do this class as bean, so we no need WebClient any more in our app except here

Reply:
made class MasterClient as bean